README.md

```diff
@@ -23,24 +22,16 @@ There are additional methods:
 
 ## Usage
 
-### Contract
+### Environment setup
 
-Set the following env vars:
+Copy the `.env.example` to `.env` and set the applicable configuration variables for the testing / deployment environment.
```

Comment:
The scripts in this project depends on some envs. 

To make it easier to use the scripts for new devs, and also to simplify the README, I added the envs in `.env.example` so people can copy it to their private `.env`

This is the approach used for example in https://github.com/cowprotocol/flash-loan-router